assets/src/edit-story/elements/text/index.js

```diff
@@ -34,8 +34,11 @@ export const defaultAttributes = {
   ...SHARED_DEFAULT_ATTRIBUTES,
   backgroundTextMode: BACKGROUND_TEXT_MODE.NONE,
   bold: false,
-  fontFamily: 'Roboto',
-  fontFallback: ['Helvetica Neue', 'Helvetica', 'sans-serif'],
+  font: {
+    family: 'Roboto',
```

Comment:
Not super relevant, but feels like this is an odd default given that we use "Open Sans" everywhere else... Feel free to ignore if you don't know any reason for this.

Reply:
Figma mentions using system font (e.g. Roboto) for new text elements, see #1204 for example.

Reply:
The default for text element has been `Roboto` though, it's just the headings that have `Open Sans` as the default.
Maybe would be good to get the default into a constant?

Reply:
> Maybe would be good to get the default into a constant?

Something we can do as part of #1204 I think.

Reply:
> Maybe would be good to get the default into a constant?
Perhaps as part of #1204?

Reply:
Moved to a constant here: 96dc787717174b744c181a98063b7f91bff6f115